datapurge/compat.py

```diff
@@ -2,7 +2,13 @@
 from functools import partial
 import django
 
-if django.VERSION >= (1,4):
+if django.VERSION >= (1,9):
+    from django.apps import apps
+    get_model = partial(apps.get_model)
+elif django.VERSION >= (1,7):
+    from django.db.models import get_model
+    get_model = partial(get_model)
```

Comment:
Frankly, it was because I was doing this relatively mechanically! I didn't even notice that I'd removed all the remaining arguments. I'll update it.